Website/DesktopModules/Hotcakes/Core/Controllers/CheckoutController.cs

```diff
@@ -104,7 +105,18 @@ public ActionResult Index()
             VerifySessionError(model);
             RenderErrorSummary(model);
             CheckFreeItems(model);
-
+            if (HccApp.CurrentStore.Settings.PaymentCreditCardGateway == "15011DF5-13DA-42BE-9DFF-31C71ED64D4A")
```

Comment:
Is there not an `enum` to use for this comparison? If not, then we should use a constant to compare so the heap doesn't have a series of new strings added to memory for a simple and frequently repeated `IF` statement.